pkgs/development/libraries/boxfort/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, cmake, pkg-config, gettext, libcsptr, dyncall
+, nanomsg, python37Packages }:
+
+stdenv.mkDerivation rec {
+  name = "boxfort";
```

Comment:
You should use `pname` and specify `unstable-2019-09-19` as a version so as to follow Nix naming conventions.